modules/junit-jupiter/src/main/java/org/testcontainers/junit/jupiter/Container.java

```diff
@@ -0,0 +1,18 @@
+package org.testcontainers.junit.jupiter;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * The {@code @Container} annotation is used in conjunction with the {@link Testcontainers} annotation
+ * to mark containers that should be managed by the Testcontainers extension.
+ *
+ * @see Testcontainers
+ * @since 1.10.0
```

Comment:
This this correct? The last version on maven central is 1.9.1 so I assumed adding a new module will cause a minor version bump.

Reply:
TBH, we don't use `@since` anywhere so far. Chances are high, that this could be release as part of 1.10.0 :smile: